RxSwiftWorkshop/Flights/ViewModel/FlightsViewModelSpec.swift

```diff
@@ -0,0 +1,91 @@
+//
+// Created by Maciej Oczko on 02.05.2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import Quick
+import Nimble
+import RxSwift
+import RxTest
+
+@testable import RxSwiftWorkshop
+
+final class FlightsViewModelSpec: QuickSpec {
+    override func spec() {
+        describe("FlightsService") {
+
+            var sut: FlightsViewModel!
+            var schipolService: FakeSchipolService!
+            var iataService: FakeIATAService!
+
+            var scheduler: TestScheduler!
```

Comment:
Implicitly Unwrapped Optionals. As a rule of thumb I don't use them, but Quick's BDD API doesn't really work well with default values. Will think of a solution, let's leave it for now